empress/recon_vis/tree.py

```diff
@@ -19,17 +24,52 @@ def __init__(self, name):
         self.parent_node = None     # Node:  parent Node or None
         self.layout = None          # NodeLayout object: layout of this node
 
+    # The @property decorator allows this to be called as .is_leaf rather than .is_leaf()
+    @property
     def is_leaf(self):
         """ returns True iff this node is a leaf/tip of the tree """
         return self.left_node is None and self.right_node is None
 
+    # The @property decorator allows this to be called as .is_root rather than .is_root()
```

Comment:
Please remove this decorator and its corresponding comments.

Reply:
That is, use `node.is_root()` instead of `node.is_root`.

Reply:
Fixed